### ISSUE TITLE
Could you please add the function of “-proxy” works by sock5, like the order of "-proxy 127.0.0.1:1080"

### ISSUE THREAD

**Issue by irEwqszdPOXlddsQSSA** [todxx/teamredminer#438]
When the mining machine works, the delay of pools could be 200 or even 300ms, sometimes you have to use proxy tools to reduce the delay, and the security and privacy of the network also can be improved. The proxy tools like sock5 are so important and useful. By using windows 10, I can use the sock5 tools. Could you please add the order of "-proxy"? So I can use the order of "-proxy 127.0.0.1:1080" to increase the speed of network. The order can be works by the local proxy soft. Some guys really need this function of "-proxy", it's so emergency.  

Thank you, brothers.

**Comment by cwaning** [comment]
Yes, this feature is really needed. It can solve many problems. I strongly hope that this feature can be added
